plugins/commands/uninstall.go

```diff
@@ -0,0 +1,56 @@
+package commands
+
+import (
+	"errors"
+	"github.com/codegangsta/cli"
+	"github.com/jfrog/jfrog-cli-core/utils/coreutils"
+	"github.com/jfrog/jfrog-cli/plugins/utils"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func UninstallCmd(c *cli.Context) error {
+	if c.NArg() != 1 {
+		return cliutils.PrintHelpAndReturnError("Wrong number of arguments.", c)
+	}
+	return runUninstallCmd(c.Args().Get(0))
+}
+
+func runUninstallCmd(requestedPlugin string) error {
```

Comment:
What is the reason for scanning the plugins directory twice?
We're scanning it once with the ```utils.GetAllPluginsNames()``` function and then again with the for loop in this function.
Can't we just find the local path of the plugin binary using the provided plugin name?

Reply:
I thought to support the command case-insensitive but I agree with you, and I think we should make all plugins names lowercase as a constraint.